CMakeLists.txt

```diff
@@ -153,9 +153,7 @@ if(BUILD_CLIENT)
     endif(APPLE)
 
     if(UNIX AND NOT APPLE)
-        find_package(INotify REQUIRED)
-    else()
-        find_package(INotify)
```

Comment:
lol ... why was this there in the first place? Good catch.

Reply:
Hysteric reasons?